exe/benchmark-driver

```diff
@@ -35,6 +35,11 @@ config = BenchmarkDriver::Config.new.tap do |c|
         executables << BenchmarkDriver::Rbenv.parse_spec(version)
       end
     end
+    o.on('--ridkuse VERSIONS', String, 'Ruby executables in ridk use (x.x.x arg1;y.y.y arg2;...) for RubyInstaller2 on Windows') do |r|
+      r.split(';').each do |version|
+        executables << BenchmarkDriver::RidkUse.parse_spec(version)
+      end
+    end
```

Comment:
Could you make this option only when `ridk` command is available, like `--timeout` option? 
(Similarly, I'll hide `--rbenv` option when rbenv is not available after merging this.)

Reply:
> I'll hide --rbenv option when rbenv is not available

done: https://github.com/benchmark-driver/benchmark-driver/commit/b2d91c6e24c5a2f82749a15138fb4409c8863d01
Sorry for causing the conflict :bow: